backend/tests/mocks/database/providermock/DBClientInterface_mock.go

```diff
@@ -0,0 +1,242 @@
+// Code generated by mockery; DO NOT EDIT.
```

Comment:
Dont we add licenses for these?

Reply:
We can. But it will be replaced each time when we generate mocks. Hence didn't try to add them.
Mockery is generating all the mocks when the `mockery` command is run